src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -533,6 +536,25 @@ message RpcLog {
   RpcLogCategory log_category = 8;
 }
 
+// Used to send metrics back to the Host 
+message RpcMetric {
+  // Matching ILogger semantics
+  // https://github.com/aspnet/Logging/blob/9506ccc3f3491488fe88010ef8b9eb64594abf95/src/Microsoft.Extensions.Logging/Logger.cs
+  // Level for the Log
+  
+  // Unique id for invocation (if exists)
+  string invocation_id = 1;
+
+  // Name of the given metric
+  string name = 2;
```

Comment:
please send a PR to https://github.com/Azure/azure-functions-language-worker-protobuf adding new message

Reply:
Does this replace the changes to the `.proto` files in this and the dotnet lang worker, then? or changes go in all 3 places?

Reply:
OK to test the changes you have with editing proto file directly in here. Please see readme in https://github.com/Azure/azure-functions-language-worker-protobuf on how to update  proto file.
- Testing with the setup you have is OK!
- When ready to check-in - Merge changes into https://github.com/Azure/azure-functions-language-worker-protobuf repo first, pull those changes into host and send separate PR - this is to make it easier to track changes to proto file that happen in host repo
- This PR should only contain code changes and no proto changes